exchange/c2cx/api.go

```diff
@@ -481,6 +482,34 @@ func (c *Client) MarketSell(symbol TradePair, amount decimal.Decimal, customerID
 	return orderID, nil
 }
 
+type GetTickerResponse struct {
```

Comment:
Make this `getTickerResponse` so it is not exported. We will return the `Data` struct instead